index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html> 
+<html lang="en-GB"> 
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Josh's Portfolio</title>
+	</head>
+		<body>
+			<header class="nav">
+				<a class="nav" href="#about">About Me</a>  
+				<a class="nav" href="#portfolio">Portfolio</a>
+				<a class="nav" href="#contact">Contact information</a>		
+			</header>
+			<section class="intro">
+				<p class="intro">HEELLL</p>
+				<p class="intro">OOOOO</p>
+				<p class="little-intro">I am Josh</p>
+				<p class="little-intro">Welcome to my world of</p>		
+				<p class="little-intro">organised chaos!</p>		
+			 </section>
+			 
+	 		<div id="about" class="about-catcher">
+				<section class="about">
+	 				<h2 class="about">About Me</h2>
+	 					<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
+						<p class="about"> I started out as an Artist beleive it or not, focused on both the buissness and creative side of the art world. My biggest passion was the 					digital arts and creating stories from a single image.</p>
+						<p class="about"> After some time in the art world and the Hospitality world, I have moved onto the path of coding.</p>
+						<p class="about"> A new way to be creative and to tell a story.</p>
+						<p class="about">A new way of organised chaos!</p>
+	 				<img class="about-img" src="joshpic.jpg" alt="A Picture of Josh">
+	 			</section>
+			</div>
+			
+			<div id="portfolio" class="portfolio-catcher">
+				<section class="portfolio-intro">
+					<h2 class="portfolio-intro"> Portfolio</h2>
+					<p class="portfolio-intro"> Here is a show case of my work so you can see how efficient and proficient yet chaoticly organised I am.</p>
+				</section>
+				</div>
+				
+				    <section class="portfolio">
+						
+					<div class="portfolio-piece1"> 
+						<main class="container">
```

Comment:
the class name container is not specific enough to know where this is on the page, there are many things that could be described as a container

Reply:
also, main is generally expected to be used only 1 time on a page, so this is more of a div